enclave_manager/Dockerfile

```diff
@@ -215,6 +215,7 @@ RUN ldconfig \
  && echo "SGX_SSL=/opt/intel/sgxssl" >> /etc/environment
 
 COPY . /project/avalon
+RUN rm -rf /project/avalon/tools/build/_dev
```

Comment:
Add comment here why we are removing it. Mention it docker build won't progress if we already has it host.

Reply:
Done